src/main/java/com/google/api/codegen/transformer/OutputTransformer.java

```diff
@@ -295,15 +296,32 @@ public void run() {
                 fieldName);
 
         type = field.getType();
-        accessors.add(context.getNamer().getFieldGetFunctionName(field));
+        accessors.add(
+            AccessorView.MemberView.newBuilder()
+                .member(context.getNamer().getFieldGetFunctionName(field))
+                .build());
       } else if (token == '[') {
         Preconditions.checkArgument(
             type.isRepeated() && !type.isMap(),
```

Comment:
Would type be a map in any case? I went though the design doc but did not find an example like that. Let's add a todo if we need to handle map types in the future.

Reply:
Ah  yes done.